srcCpp/RTIDDSImpl.cxx

```diff
@@ -2256,15 +2256,28 @@ bool RTIDDSImpl<T>::Initialize(int argc, char *argv[])
                 "dds.domain_participant.auto_throttle.enable", "true", false);
     }
 
+    // Registered _loggerDevice
```

Comment:
Move to top of the function (before all the RTI DDS calls.

Reply:
Done